apps/service-portal/src/environments/environment.prod.ts

```diff
@@ -13,17 +15,10 @@ export default {
       'https://3c45a55273774b91a897b85e0a1243d1@o406638.ingest.sentry.io/5501494',
   },
   documentProviderAdmins: process.env.NX_DOCUMENT_PROVIDER_ADMINS || '',
-  featureFlags: {
-    applications: false,
-    documents: true,
-    settings: true,
-    finance: true,
-    family: true,
-    health: false,
-    education: false,
-    delegation: false,
-    assets: false,
-    drivingLicense: false,
-    documentProvider: false,
-  },
+  featureFlagSdkKey:
```

Comment:
Erm... really? can't we somehow get this from the environment? You're at least missing `www.island.is`... but I don't want to maintain a list of possible ingress hosts in code...

Reply:
I don't like this either... I am up for any ideas on how we could do this.
The service portal is a react project which uses the docker-static configuration.
I am not aware of any way to get values from environment when working with docker-static.

Is there something that I am missing?

Reply:
Some kind of env.json that we can generate on container startup via envsubst? And then fetch it on application startup? Either way it's not acceptable to *not* have these capabilities.

Reply:
I thought this was on your backlog, but I didn't find an issue for this.
I created #2972